httpclient5/src/test/java/org/apache/hc/client5/http/psl/TestPublicSuffixMatcher.java

```diff
@@ -70,6 +70,8 @@ public void testGetDomainRoot() throws Exception {
         Assert.assertEquals("blah.ac.jp", matcher.getDomainRoot("blah.blah.ac.jp"));
         Assert.assertEquals("garbage", matcher.getDomainRoot("garbage"));
         Assert.assertEquals("garbage.garbage", matcher.getDomainRoot("garbage.garbage"));
+        Assert.assertEquals("garbage.garbage", matcher.getDomainRoot("garbage.garbage.garbage.garbage"));
+        Assert.assertEquals("garbage.garbage", matcher.getDomainRoot("*.garbage.garbage"));
```

Comment:
@jeromedemangel I do not think this is correct. The registerable part of `garbage.garbage` should be `garbage`. Please see b6c9688.